### ISSUE TITLE
[Bug]: [PartitionKey] error msg is a bit confusing when creating a schema with 2 fields as is_partition_key=True

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#24149]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.2.0-20230515-8b8f9f1e
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus-2.2.9.dev8
```

### Current Behavior

 message=Partition key field can only be one.
I think it is a bit confusing 

### Expected Behavior

message=There can only be one partition key field.
might be better

### Steps To Reproduce

```markdown
1. create a schema with two fields set is_partition_key=True
```

**Comment by xige-16** [comment]
primary field has the same format error 
```
<PrimaryKeyException: (code=1, message=Primary key field can only be one.)>
```

**Comment by xige-16** [comment]
> ### Is there an existing issue for this?
> * [x]  I have searched the existing issues
> 
> ### Environment
> ```gfm
> - Milvus version: 2.2.0-20230515-8b8f9f1e
> - Deployment mode(standalone or cluster):
> - MQ type(rocksmq, pulsar or kafka):    
> - SDK version(e.g. pymilvus v2.0.0rc2): pymilvus-2.2.9.dev8
> ```
> 
> ### Current Behavior
> message=Partition key field can only be one. I think it is a bit confusing
> 
> ### Expected Behavior
> message=There can only be one partition key field. might be better
> 
> ### Steps To Reproduce
> ```gfm
> 1. create a schema with two fields set is_partition_key=True
> ```
> 
> ### Milvus Log
> _No response_
> 
> ### Anything else?
> _No response_

@XuanYang-cn  which one is better?

**Comment by xige-16** [comment]
change to error mdg to message=Expected only one partition key field, got [field1, field2, ...].)>

**Comment by yanliang567** [comment]
verified fix on  2.2.0-20230517-d4f7ebd1 and pymilvus-2.2.9.dev11
